specification/api-tracing.md

```diff
@@ -308,7 +308,8 @@ description](overview.md#links-between-spans).
 A `Link` is defined by the following properties:
 
 - (Required) `SpanContext` of the `Span` to link to.
-- (Optional) One or more `Attribute`.
+- (Optional) One or more `Attribute`s with the same semantics as defined for
```

Comment:
I don't think there is an `Attribute` entity defined anywhere, so just "attributes" would be fine, but that's for another PR.